app/assets/javascripts/reducers/users.js

```diff
@@ -23,12 +23,22 @@ const SORT_DESCENDING = {
 
 export default function users(state = initialState, action) {
   switch (action.type) {
-    case RECEIVE_USERS: return {
-      ...state,
-      users: action.data.course.users,
-      isLoaded: true,
-      lastRequestTimestamp: Date.now()
-    };
+    case RECEIVE_USERS: {
+      const initialSortKey = 'real_name';
+      const sortKey = state.sort.sortKey || initialSortKey;
```

Comment:
Is there any reason the `real_name` value can't simply be set in the `initialState`?

Reply:
Updated.